### DISCUSSION TITLE
changing top left icon

### DISCUSSION THREAD

**Discussion by realrevy223** [Shina-SG/Shina-Fox#38] [Q&A]
hi! i would like to change the frieren icon in the top left. is there any way to do so?

**Comment by realrevy223** [comment]
"dark.png" and "light.png" are the icons...

**Comment by antonioszm** [comment]
If you simply want to overwrite the frieren icon, take your icon 106px106p or smaller with the name dark.png (or light.png if you are using the light theme) and place it in the chrome folder of your Root Directory, and you are ready

If you don't want to delete the frieren image from your chrome folder in userChrome.css on line 451-461 this is the path to the photo, you will need to add its icon to the chrome folder and change the url of the example image

```
@media (prefers-color-scheme: dark) {
   .tab-icon-image {
     content: url('myimage.png') !important;
   }
 }
```
